### DISCUSSION TITLE
LR2-style tags

### DISCUSSION THREAD

**Discussion by LordVysh** [wcko87/beatoraja-english-guide#40] [beatoraja questions / troubleshooting]
Is it possible to tag charts like in LR2? I used to tag all the charts I liked with a specific tag so I didn't have to go searching for them.

**Comment by wcko87** [comment]
There isn't a tag system in beatoraja yet (I plan to implement one sometime in the future), but there is the custom tables feature, which I currently use to "tag" charts.

See: https://github.com/wcko87/beatoraja-english-guide/discussions/26#discussioncomment-1457168
It's not ideal, but it works well enough for bookmarking charts.

You can also favorite songs/charts with F8/F9.
